spec/yaml/swiftnav/sbp/bootload.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Fergus Noble <fergus@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.bootload
+description: |
+  Messages for the bootloading configuration on the Piksi. These are
+  in the implementation-defined range (0x0000-0x00FF), and intended
+  for internal-use only. Note that some of these messages taking a
+  request from a host and a response from the Piksi share the same
+  message type ID.
+stable: False
+public: True
+include:
+  - types.yaml
+definitions:
+
+ - MSG_BOOTLOADER_HANDSHAKE:
+    id: 0x00B0
+    public: True
+    short_desc: Bootloading handshake (Host <=> Piksi).
+    desc: |
+      The host issues a handshake message prior to flashing Piksi's
+      firmware.
+    fields:
+      - handshake:
+          type: u8
+          desc: Handshake value
+
+ - MSG_BOOTLOADER_JUMP_TO_APP:
+    id: 0x00B1
+    public: True
+    short_desc: Bootloader jump to application (Host => Piksi)
+    desc: |
+      The host initiates the bootloader to jump to the application.
+    fields:
+      - jump:
+          type: u8
+          desc: Ignored by the Piksi.
+
+ - MSG_NAP_DEVICE_DNA:
+    id: 0x00DD
+    public: True
+    short_desc: Send FPGA device DNA over UART (Host <=> Piksi).
+    desc: |
+      The device DNA message from the host reads the authentication
+      hash, the unique device DNA, from the Swift Navigation
+      Acceleration Peripheral (SwiftNAP), a Spartan 6 FPGA. By
+      convention, the host message buffer is empty; the Piksi returns
+      the device DNA in a MSG_NAP_DEVICE_DNA message.
+    fields:
+      dna:
+        type: array
+        fill: u8
+        size: 8
+        desc: 57-bit SwiftNAP FPGA Device DNA
```

Comment:
Just reads the DNA, doesn't do anything with the authentication hash in the configuration flash.